dubbo-cluster/src/main/java/org/apache/dubbo/rpc/cluster/support/migration/MigrationRule.java

```diff
@@ -82,9 +83,9 @@ public static MigrationRule queryRule() {
         return parse(rawRule);
     }
 
-    private  static MigrationRule getMigrationRule(String step) {
+    private static MigrationRule getMigrationRule(String step) {
         MigrationRule rule = new MigrationRule();
-        rule.setStep(Enum.valueOf(MigrationStep.class, StringUtils.isBlank(step) ? MigrationStep.APPLICATION_FIRST.name() : step));
+        rule.setStep(Enum.valueOf(MigrationStep.class, StringUtils.isBlank(step) ? MigrationStep.FORCE_INTERFACE.name() : step));
```

Comment:
The APPLICATION_FIRST is ok, I think. The logic in org.apache.dubbo.registry.client.migration.MigrationInvoker#migrateToServiceDiscoveryInvoker will handle service discovery and interface discovery both.

Reply:
The application-level service discovery is incomplete in 2.7.x, maybe we should use stable version(FORCE_INTERFACE) here. But keep APPLICATION_FIRST in 3.0, but also we should discuss this in more detail. And I'll extract other changes to a separated pull request and keep this here.